src/p4d.js

```diff
@@ -1,20 +1,18 @@
 import { auth, refresh } from './auth'
 
-import { createReadStream, statSync } from 'fs'
+import { readFileSync, statSync } from 'fs'
 import { request } from 'https'
 import { basename } from 'path'
 
 const FormData = require('form-data')
 
-async function doit (gameId, filename, name, notes, config) {
+async function doit (gameId, filename, name, notes, makePublic, config) {
```

Comment:
The argument list is getting pretty long. We need to refactor that with a dictionary (maybe not in this PR though)